### DISCUSSION TITLE
How to keep former screening decisions when starting a new project in ASReview?

### DISCUSSION THREAD

**Discussion by Tsklopper** [asreview/asreview#559] [Q&A]
I already started labeling before I came across ASReview. How can I keep my former screening decisions when starting a new project in ASReview?

**Comment by SagevdBrand** [comment]
Excellent question!

You could simply add a column (in excel for example) called ‘included’ or ‘label_included’ to the file you want to upload to ASReview. In this column, you can indicate previous screening decisions on records with 0s (irrelevant) and 1s (relevant). ASReview will use this information to train the model. 

I think this is  also related to the question posed in #562, where the answer provided by @terrymyc  was as follows:

> You can export the results of ASReview you have so far and add your new publications to this file. Make sure that for your new publications the column included is empty and rename or delete the column of record_id. This latter will save you from running into errors. Then simply import this updated file to ASReview and you can continue the screening process. 

In case you would want to add records to your screening-process as extra prior knowledge for example, you could combine these two answers!
